gradlew.bat

```diff
@@ -14,7 +14,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=-Xmx128m -Dfile.encoding=UTF-8
+set DEFAULT_JVM_OPTS=-Xmx64m -Dfile.encoding=UTF-8

```

Comment:
When I generate a new wrapper with the latest nightly on release now, I get this generated in gradlew.bat:
`set DEFAULT_JVM_OPTS=-Xmx128m -Dfile.encoding=UTF-8"-Xmx64m"`

And this in `gradlew`:
`DEFAULT_JVM_OPTS='"-Xmx64m"'`

The gradlew script seems to work. I haven't tried the gradlew.bat script, but it looks wrong.

I think DEFAULT_JVM_OPTS in gradlew script shouldn't be singled quoted.  

I think the problems with our wrapper script are self-inflicted somewhere, maybe `WrapperPlugin.kt`?

Reply:
Interesting, I didn't change anything about the quoting, so if this is a problem it would have existed in our script generator for a long time.